src/mlang/backend_ir/bir_interpreter.mli

```diff
@@ -118,19 +120,45 @@ module type S = sig
   (** Before execution of the program, replaces the [undefined] stubs for input
       variables by their true input value *)
 
+  val print_output : Bir_interface.bir_function -> ctx -> unit
+
   val raise_runtime_as_structured : run_error -> ctx -> Mir.program -> 'a
   (** Raises a runtime error with a formatted error message and context *)
+
+  val evaluate_expr : ctx -> Mir.program -> Bir.expression Pos.marked -> value
+
+  val evaluate_program : Bir.program -> ctx -> int -> ctx
 end
 
-module RegularFloatInterpreter : S
+module FloatDefInterp : S
+
+module FloatMultInterp : S
+
```

Comment:
An interpretation guide would be handy here to know how to decipher the names of those interpreters :)